src/swap/components/SwapSettingsSlippageInput.tsx

```diff
@@ -0,0 +1,67 @@
+import { type ChangeEvent, useCallback, useEffect, useState } from 'react';
+import { cn } from '../../styles/theme';
+import type { SwapSettingsSlippageInputReact } from '../types';
+import { useSwapContext } from './SwapProvider';
+
+export function SwapSettingsSlippageInput({
+  className,
+  defaultSlippage = 3,
+  customSlippageEnabled,
```

Comment:
is this a boolean? if that is the case we should start this prop with `is`

Reply:
yes it's a boolean, will update